examples/esp_idf/README.md

```diff
@@ -16,7 +16,7 @@ For Linux users, you can install esp-idf with these commands:
 sudo apt-get install git wget flex bison gperf python3 python3-venv cmake ninja-build ccache libffi-dev libssl-dev dfu-util libusb-1.0-0
 mkdir -p ~/esp
 cd ~/esp
-git clone --recursive https://github.com/espressif/esp-idf.git -b v5.2
+git clone --recursive https://github.com/espressif/esp-idf.git -b v5.2.1
```

Comment:
You didn't touch it, but line 75 of the README has a space after the period which is why the pre-commit failed: 

https://github.com/golioth/golioth-firmware-sdk/blob/0fcf3802ebc18b033d1efc5fb9504be20c4b7035/examples/esp_idf/README.md?plain=1#L75